chargeOclock/Source/Other/NetworkAPI.swift

```diff
@@ -19,6 +19,12 @@ extension NetworkAPI {
 	func delete(client: Client, completion: @escaping NetworkCompletion) {
 		performDeleteRequest(to: "clients/\(client.id)", completion: completion)
 	}
+
+	func addClient(name: String, completion: @escaping NetworkCompletion) {
+		let header = ["Content-Type" : "application/json; charset=utf-8"]
```

Comment:
since all API requests will have this header set, it would make sense to move this further down the line